src/sass/user-profile.scss

```diff
@@ -77,31 +83,40 @@
     }
   }
 
-  .saved-form-title {
+  .saved {
     color: $color-primary;
   }
 
+  .form-title {
+    @include media ($medium-large-screen) {
+    padding-right: 24px;
+    }
+  }
+
+  .remove-saved-application-button {
+    float: right;
+  }
+  
   .remove-saved-application-container {
     margin-bottom: 16px;
     float: right;
-    @include media($medium-large-screen) {
-      float: left;
-    }
+    padding-left: 8px;
```

Comment:
It doesn't have to be in this PR, but I think making the markup for the button row use flexbox instead of grid classes and floats would let you clean up some of the styles.

Reply:
Brilliant idea, @jbalboni.